subsys/nrf_security/src/CMakeLists.txt

```diff
@@ -265,4 +265,4 @@ endif()
 if (NOT BUILD_INSIDE_TFM)
   # Add zephyr-specific code
   add_subdirectory(zephyr)
-endif()
+endif()
```

Comment:
needs a newline

Reply:
This I cannot do it seems, compliance complains.